circle.yml

```diff
@@ -1,11 +1,18 @@
 dependencies:
   pre:
     - npm link
+    - echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
 
 test:
   override:
     - fliplet env staging
     - fliplet login -u $TEST_USER -p $TEST_PASSWORD
     - fliplet organization $TEST_ORGANIZATION
     - fliplet create-widget awesome-widget
-    - cd awesome-widget && fliplet test
\ No newline at end of file
+    - cd awesome-widget && fliplet test
+
+deployment:
+  npm:
```

Comment:
makes sense, please add the steps to create a release on the readme so we know how to publish to npm.